tests/sharingTest.js

```diff
@@ -232,95 +600,80 @@ describe('Main: Currently testing file/folder sharing,', function () {
           })
         })
 
-        it('checking method : getShares for shared file', async function () {
```

Comment:
is this test removed?

Reply:
isn't is that here https://github.com/owncloud/owncloud-sdk/pull/728/files#diff-3e96a43471ac474278d7585f7a1ab0a201ce7c5a935710b88c2a9304710d8a76R763 ?

Reply:
that test is in this line https://github.com/owncloud/owncloud-sdk/blob/a4355e367127b9412421e5d8a193c6499c0757f0/tests/sharingTest.js#L763